cognito-pre-token-generation-openid/src/main/java/no/unit/nva/cognito/UserSelectionUponLoginHandler.java

```diff
@@ -51,159 +51,143 @@
-
+    
         var authenticationInfo = collectAuthenticationInformation(nin, orgFeideDomain, personFeideIdentifier);
         final var usersForPerson = userCreator.createUsers(authenticationInfo);
-
+    
         final var roles = rolesPerCustomer(usersForPerson);
         authenticationInfo.updateCurrentCustomer();
         authenticationInfo.updateCurrentUser(usersForPerson);
-
-        final var accessRights = createAccessRights(usersForPerson);
+    
+        final var accessRights = createAccessRightsPerCustomer(usersForPerson);
         updateCognitoUserAttributes(input, authenticationInfo, accessRights, roles);
         injectAccessRightsToEventResponse(input, accessRights);
         return input;
     }
-
+    
     private static String extractNin(Map<String, String> userAttributes) {
         return Optional.ofNullable(userAttributes.get(NIN_FOR_FEIDE_USERS))
-            .or(() -> Optional.ofNullable(userAttributes.get(NIN_FON_NON_FEIDE_USERS)))
-            .orElseThrow();
+                   .or(() -> Optional.ofNullable(userAttributes.get(NIN_FON_NON_FEIDE_USERS)))
+                   .orElseThrow();
     }
-
+    
     private static String extractOrgFeideDomain(Map<String, String> userAttributes) {
         return Optional.ofNullable(userAttributes.get(ORG_FEIDE_DOMAIN)).orElse(null);
     }
-
+    
     private static String extractFeideIdentifier(Map<String, String> userAttributes) {
         return Optional.ofNullable(userAttributes.get(FEIDE_ID)).orElse(null);
     }
-
+    
     @JacocoGenerated
     private static CognitoIdentityProviderClient defaultCognitoClient() {
         return CognitoIdentityProviderClient.builder()
-            .credentialsProvider(EnvironmentVariableCredentialsProvider.create())
-            .httpClient(UrlConnectionHttpClient.create())
-            .region(AWS_REGION)
-            .build();
+                   .credentialsProvider(EnvironmentVariableCredentialsProvider.create())
+                   .httpClient(UrlConnectionHttpClient.create())
+                   .region(AWS_REGION)
+                   .build();
     }
-
+    
     private AuthenticationInformation collectAuthenticationInformation(String nin,
                                                                        String orgFeideDomain,
                                                                        String personFeideIdentifier) {
         return attempt(() -> new NationalIdentityNumber(nin))
-            .map(idNumber -> userCreator.collectPersonInformation(idNumber, personFeideIdentifier, orgFeideDomain))
-            .map(AuthenticationInformation::new)
-            .orElseThrow();
+                   .map(idNumber -> userCreator.collectPersonInformation(idNumber, personFeideIdentifier,
+                       orgFeideDomain))
+                   .map(AuthenticationInformation::new)
+                   .orElseThrow();
     }
-
-    private List<String> createAccessRights(List<UserDto> usersForPerson) {
```

Comment:
remove method after removing the legacy functionality of adding `USER@<customerId>` in the cognito groups for the e2e testing. Now we are using a dedicated endpoint for e2e tests